index.html

```diff
@@ -1,17 +1,196 @@
+          <h2>Zsh</h2>
+          <span class="material-icons"><a href="#">east</a></span>
+        </div>
+        <div class="menu-tab-links">
+          <h2>Interfaces</h2>
+          <span class="material-icons"><a href="#">east</a></span>
+        </div>
+      </div>
+    </div>
+    <!-- Mobile Home page -->
+    <div id="bg-menu" class="bg-menu">
+      <button id="bg-menu-btn" type="button">
+        <span class="btn-line"></span>
+        <span class="btn-line"></span>
+        <span class="btn-line"></span>
+      </button>
+      <h2>"Make things possible"</h2>
+    </div>
+
+    <header id="bg-header">
+      <h2 class="bgh2color">What makes technology possible?</h2>
+      <div class="description-header">
+        How the hardware gets connected with the software? This like other important questions we need to have in our
+        minds at the moment to develop something related to tech.
+      </div>
+      <div class="date">
+        <h2>2021.10.15(THU) ~ 16(FRI)</h2>
+        <p>
+          @ Tech, art and more
+        <p>
+      </div>
+    </header>
+    <!-- Mobile About -->
+    <main id="about">
+      <div>
+        <h1 class="about-header">Main Introduction</h1>
+        <div class="red-line"></div>
+      </div>
+
+      <div class="cards-container">
+        <div class="card">
+          <ul>
+            <li>
+              <i class="fas fa-cogs"></i>
+            </li>
+          </ul>
+          <h2>Assembler</h2>
+          <p>
+            This language stay more close with the hardware than anyone.
+          </p>
+        </div>
+        <div class="card">
+          <ul>
+            <li>
+              <i class="fab fa-contao"></i>
+            </li>
+          </ul>
+          <h2>C.</h2>
+          <p>
+            This one is also close to the hardware but not more that assembler.
+          </p>
+        </div>
+        <div class="card">
+          <ul>
+            <li>
+              <i class="fab fa-js"></i>
+            </li>
+          </ul>
+          <h2>JavaScript</h2>
+          <p>
+            The tool that all webdev wants to master.
+          </p>
+        </div>
+        <div class="card">
+          <ul>
+            <li>
+              <i class="fas fa-terminal"></i>
+            </li>
+          </ul>
+          <h2>Zsh</h2>
+          <p>
+            Master Zsh is essential to work more faster.
+          </p>
+        </div>
+        <div class="card">
+          <ul>
+            <li>
+              <i class="fas fa-desktop"></i>
+            </li>
+          </ul>
+          <h2>Interface</h2>
+          <p>
+            A tool that help every people manage their computer.
+          </p>
+        </div>
+      </div>
+      <button type="button">Join to see more</button>
+      <ul class="link-below">
+        <li><a href="#">SEE THE WHOLE PROGRAM</a></li>
+      </ul>
+    </main>
+    <div id="feature-speakers-container"></div>
+
+    <footer>
```

Comment:
- [ ] **OPTIONAL**
It seems that the footer doesn't show up on mobile screen homepage, only the about page, I highly recommend fixing this issue since it is mandatory and should be mentioned in the earlier review.

Reply:
It is only for the desktop version and the about section on mobile.